cschwitalla/Ligandomics_Analysis/Ligandomics_Analysis.R

```diff
@@ -434,6 +442,16 @@ ggVennDiagram(set_filter, label_alpha = 0, label = "count", label_size = 3) + sc
 ggVennDiagram(setII_filter, label_alpha = 0, label = "count", label_size = 3) + scale_fill_gradient(low = "papayawhip", high = "paleturquoise4")+ scale_color_manual(values = c(NEC = "grey", T1 = "grey", INF = "grey", BEN = "grey")) + ggtitle("Class II protein acc, beningiome filterd")
 
 
+
+set_filter_seq = list(NEC = NEC_I_uniqe_acc_filterd$Sequence, T1 = T1_I_uniqe_acc_filterd$Sequence, INF = INF_I_uniqe_acc_filterd$Sequence, BEN = Benigniome_I_unique_acc$Peptide, BENDB = c(Benigniome_I_unique_acc$Acc,HLA_Atlas_I_uniqaccs$peptide_sequence))
+setII_filter_seq = list(NEC = NEC_II_uniqe_acc_filterd$Sequence, T1 = T1_II_uniqe_acc_filterd$Sequence, INF = INF_II_uniqe_acc_filterd$Sequence, BEN = Benigniome_II_unique_acc$Peptide, BENDB = c(Benigniome_II_unique_acc$Acc,HLA_Atlas_II_uniqaccs$peptide_sequence))
+
+
+ggVennDiagram(set_filter_seq, label_alpha = 0, label = "count", label_size = 3) + scale_fill_gradient(low = "papayawhip", high = "paleturquoise4")+ scale_color_manual(values = c(NEC = "grey", T1 = "grey", INF = "grey", BEN = "grey")) + ggtitle("Class I peptides")
```

Comment:
You can also make a function that you call twice, once with the class 1 and then class 2